config/common.py

```diff
@@ -80,7 +80,6 @@
             'name': 'maintenance',
             'options': [
                 '--unload-module=reframe',
-                '--exec-policy=async',
```

Comment:
are you removing this (also for production) just for cleaning up, because it is the default exec policy?

Reply:
Yes, it has been the default for a long time